test/vault_main_test.go

```diff
@@ -57,45 +71,47 @@ var testCases = []testCase{
 
 func TestMainVaultCluster(t *testing.T) {
 	t.Parallel()
-	amiIds := map[string]string{}
 
 	test_structure.RunTestStage(t, "setup_amis", func() {
-		awsRegion := aws.GetRandomRegion(t, nil, nil)
-		test_structure.SaveString(t, WORK_DIR, SAVED_AWS_REGION, awsRegion)
-
 		tlsCert := generateSelfSignedTlsCert(t)
 		saveTlsCert(t, WORK_DIR, tlsCert)
 
-		var identifierToOptions = map[string]*packer.Options{
-			"vaultEnterpriseUbuntu":      composeAmiOptions(t, AMI_EXAMPLE_PATH, "ubuntu16-ami", tlsCert, awsRegion, getUrlFromEnv(t)),
-			"vaultEnterpriseAmazonLinux": composeAmiOptions(t, AMI_EXAMPLE_PATH, "amazon-linux-ami", tlsCert, awsRegion, getUrlFromEnv(t)),
-			"vaultUbuntu":                composeAmiOptions(t, AMI_EXAMPLE_PATH, "ubuntu16-ami", tlsCert, awsRegion, ""),
-			"vaultAmazonLinux":           composeAmiOptions(t, AMI_EXAMPLE_PATH, "amazon-linux-ami", tlsCert, awsRegion, ""),
+		amisPackerOptions := map[string]*packer.Options{}
```

Comment:
Instead of having all amis in the same region, each gets a random region now

Reply:
Is this to avoid limits in a region?

Reply:
yes